.github/workflows/maven.yml

```diff
@@ -14,14 +14,20 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
Since this is github actions for v2, should we omit v1.x?

Reply:
makes sense, done